osis-core/src/main/java/com/scality/osis/utils/ScalityModelConverter.java

```diff
@@ -326,6 +343,24 @@ private static OsisTenant toOsisTenant(AccountData account) {
                 .tenantId(account.getId());
     }
 
+    /**
+     * Converts IAM User object to OSIS User object
+     *
+     * @return the osis User
+     */
+    public static OsisUser toOsisUser(User user, String tenantId) {
+        return new OsisUser()
+                .cdUserId(user.getUserName())
+                .canonicalUserId(user.getUserName())
+                .userId(user.getUserName())
```

Comment:
Should these be `user.getCanonicalId` and `user.getUserId` respectively?

Reply:
* For OSIS we are assigning `canonicalUserId` with `cd_user_id` i.e., Vault `username` so we can implement `getUser` using `canonicalUserId` on OSIS. (ref: [design for canonical user ID](https://github.com/scality/vmware-ose-scality/blob/documentation/S3C-4180-ose-design-2/Design.md#get-user-w-canonical-id)
* Regarding `userId`, vault `username` is mapped to OSIS `userID`. (Ref: [design for `get-user`](https://github.com/scality/vmware-ose-scality/blob/documentation/S3C-4180-ose-design-2/Design.md#get-user-w-canonical-id)

Reply:
I understand now.
Thanks.